aries-site/src/examples/foundation/icons/IconCoreExample.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Card,
+  CardHeader,
+  CardBody,
+  Grid,
+  Heading,
+  Paragraph,
+  Text,
+} from 'grommet';
+import {
+  Add,
+  Close,
+  Download,
+  Edit,
+  Filter,
+  FormDown,
+  FormUp,
+  Notification,
+  Search,
+  Subtract,
+  Trash,
+} from 'grommet-icons';
+
+const CoreIcon = ({ Icon, name, aliases, guidance }) => (
+  <Card elevation={false}>
+    <CardHeader
+      direction="column"
+      align="stretch"
+      gap="small"
+      pad={{ top: 'medium', horizontal: 'medium', bottom: 'small' }}
+    >
+      <Icon />
+      <Box>
+        <Heading level="3" size="small" margin="none">
+          {name}
+        </Heading>
+        {aliases && (
+          <Box direction="row" gap="xsmall" wrap>
+            <Text>or</Text>
+            {aliases.map((a, i) => (
+              <>
+                <Text weight="bold">{a}</Text>
```

Comment:
Aligning font colors of Heading and aliases. Soft opinion.

```suggestion
                <Text weight="bold" color="text-strong" >{a}</Text>
```